packages/web-react/src/components/Button/README.md

```diff
@@ -21,22 +21,36 @@ import { Button } from '@lmc-eu/spirit-web-react';
-| `type`       | `string`                                                                                  | `button`  | no       | Type of the Button                                   |
-
-For more information see [Button] component.
+| Prop name     | Type                                                                                      | Default    | Required | Description                                          |
+| ------------- | ----------------------------------------------------------------------------------------- | ---------- | -------- | ---------------------------------------------------- |
+| `children`    | `ReactNode`                                                                               | `null`     | no       | Content of the Button                                |
+| `color`       | [Action Color dictionary][dictionary-color], [Emotion Color dictionary][dictionary-color] | `primary`  | no       | Color variant                                        |
+| `elementType` | `ElementType`                                                                             | `'button'` | no       | Type of element                                      |
+| `isBlock`     | `bool`                                                                                    | `false`    | no       | Span the element to the full width of its parent     |
+| `isDisabled`  | `bool`                                                                                    | `false`    | no       | If true, Button is disabled                          |
+| `isSquare`    | `bool`                                                                                    | `false`    | no       | If true, Button is square, usually only with an icon |
+| `name`        | `string`                                                                                  | -          | no       | For use a button as a form data reference            |
+| `onClick`     | `string`                                                                                  | `null`     | no       | JS function to call on click                         |
+| `ref`         | `ForwardedRef<HTMLButtonElement>`                                                         | -          | no       | Button element reference                             |
+| `size`        | [Size dictionary][dictionary-size]                                                        | `medium`   | no       | Size variant                                         |
+| `type`        | `string`                                                                                  | `button`   | no       | Type of the Button                                   |
+
+For more information see [Button] component. Button also contain all the appropriate
+attributes according to the type of element. The default value for Button is `<button>`.
```

Comment:
> The default value for Button is `<button>`.

I don't follow, what do you mean by this?

Reply:
Default element type?

Reply:
fixed